src/users/service/userService.ts

```diff
@@ -34,8 +34,17 @@ export class UsersService {
     }
   }
 
-  async findAll(): Promise<UserPivot[]> {
-    return await this.usersRepository.findAll();
+  async findAll(
+    limit: number = 10,
+    page: number = 0
+  ): Promise<{ data: User[]; count: number }> {
+    if (limit > 100) {
+      throw {
+        statusCode: 400,
+        message: 'Pagination limit exceeded'
+      };
```

Comment:
Are you sure it is throwing 400 error code? Is this not found exception, bad request or what?